erpnext/public/js/controllers/transaction.js

```diff
@@ -866,6 +867,10 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 				refresh_field('to_date');
 			}
 		}
+	},
+	
+	gross_profit: function(item) {
```

Comment:
@nabinhait 

Not getting value of base_rate fields from js side on rate trigger if transaction is in company currency.